### PR TITLE
feat(unlock-app) - show metadata in keychain page

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
Add ability to key owner to see details of metadata

# Issues

Fixes #
Refs #

#### add new icon in key card with tooltip "show metadata"
Screenshot 2022-07-26 at 13 28 06

#### metadata modal with list of values
Screenshot 2022-07-26 at 13 22 48

#### metadata modal loading state
Screenshot 2022-07-26 at 13 21 27

#### metadata modal with no values
Screenshot 2022-07-26 at 13 24 06

### double check metadata modal have same data of keyMember page 

Screenshot 2022-07-26 at 18 35 50
Screenshot 2022-07-26 at 18 17 43

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/keychain/KeyMetadataModal.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                    key={index}
+                    {...register(key, {
+                      value,
+                    })}
+                  />
+                )
+              })}
+            </form>
+          ) : (
+            <div>There is not metadata</div>
```
Comment (julien51) [inline unlock-app/src/components/interface/keychain/KeyMetadataModal.tsx:84]:
```suggestion
            <div>There is no metadata associated with that key.</div>
```

**Review by julien51 — COMMENTED** [review]
Please follow @searchableguy 's advice but that's a good start!

File: unlock-app/src/components/interface/keychain/KeyMetadataModal.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      ) : (
+        <div className="flex flex-col gap-3 p-4 min-h-[300px] text-left">
+          <span className="font-semibold text-lg">
+            {`${lock.name} - Metadata`}{' '}
+          </span>
+          {hasValues ? (
+            <form action="">
+              {values?.map(([key, value], index) => {
+                return (
+                  <Input
```
Comment (julien51) [inline unlock-app/src/components/interface/keychain/KeyMetadataModal.tsx:77]: I assume the use of `Input` is to make it easy to update later on?

**Review by julien51 — APPROVED** [review]
LG

File: unlock-app/src/components/interface/keychain/KeyMetadataModal.tsx (hunk 2 above)
Comment (kalidiagne) [reply unlock-app/src/components/interface/keychain/KeyMetadataModal.tsx:77]: yes! now Input is read-only, thinking about the ability to edit metadata, that will be added next, we have all in place

**Pull request merged**